src/lib/moderationList.ts

```diff
@@ -0,0 +1,21 @@
+import { readFileSync } from 'fs';
+import path from 'path';
+
+const FIELDS = ['flaggedLinks', 'flaggedProposalIds', 'verifiedSpaces'];
```

Comment:
maybe can be just `flaggedProposals`

Reply:
We can, if we're sure that we will not have any other types of flagged proposal type in the future

Reply:
we can add a suffix for future ones, for ids we can call them just proposals